server/schema/clams.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE clams IF NOT EXISTS{
+    clamId SERIAL PRIMARY KEY,
+    rakerId INTEGER NOT NULL,
+    lat DECIMAL NOT NULL,
+    long DECIMAL NOT NULL,
+    length INTEGER NOT NULL,
```

Comment:
`length`, `width`, `weight` might be better off as decimals as opposed to integers